torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import tempfile
 import time
+import pickle
```

Comment:
nit: sort

Reply:
I've removed this import, in the updated version it is unnecessary